test/integration/patients/patient/patient-flow.js

```diff
@@ -340,10 +330,52 @@ context('patient flow page', function() {
       .find('[data-activity-region]')
       .find('.comment__item')
       .last()
+      .as('socketComment')
       .should('contain', formatDate(testTs(), 'AT_TIME'))
-      .should('contain', 'Test Clinician')
+      .should('contain', 'Clinician McTester')
       .should('contain', 'New websocket comment.');
 
+    cy
+      .get('@socketComment')
+      .find('.js-edit')
+      .click();
```

Comment:
Not sure if this is 100% necessary. But I want to include something in the tests that signifies that if the comment is in a "editing" state, the `CommentEdited` ws notification will re-render the comment. And potentially wipe out the user's edits if they have one.

Since a comment can only be edited by it's author, this is an edge case that should only happen when a user is editing a form in two different browser tabs or devices at the same time.